### DISCUSSION TITLE
Is there a Youtube revanced app ?

### DISCUSSION THREAD

**Discussion by Adithyan007hack** [ReVanced/revanced-discussions#854] [Question] [answered]
Is there a Youtube revanced standalone app other than the Revanced manager
as listed in this website https://revanced.io/ is it safe to download ?
the developer is oSumAtrIX is he legit ?

**Comment by reisxd** [comment, marked as answer]
ReVanced.io is not the official site.

**Reply by Adithyan007hack** [reply]
Is the app Youtube Revanced listed in the website safe?

**Reply by reisxd** [reply]
No.

**Reply by Adithyan007hack** [reply]
Then why the developer name is oSumAtrIX in that website 
and he also a member of this revanced repo

**Reply by reisxd** [reply]
> Then why the developer name is oSumAtrIX in that website 
> and he also a member of this revanced repo

Again, it's not the official website.

**Reply by Adithyan007hack** [reply]
Is oSumAtrIX making an unofficial Revanced app other than this

**Reply by reisxd** [reply]
Again, the site is not owned by anyone in the team and is unofficial.

**Reply by Adithyan007hack** [reply]
but all your names are listed on it 
![2023-01-01_20-59](https://user-images.githubusercontent.com/87800041/210176251-b031a201-011c-4e79-9136-1bc8954a0b6b.png)

**Reply by CnC-Robert** [reply]
So? its a fake site

**Reply by Adithyan007hack** [reply]
Why would someone create a fake website without any profit for him even the donate page in that fake website leads to the official Revanced donate page .

**Reply by Ushie** [reply]
Because they wanted to, we don't really know why, but we have said and won't say again from here that it is unofficial and nobody listed in there has anything to do with the revanced.io

The only official website is anything under the revanced.app domain such as https://revanced.app/

**Reply by LisoUseInAIKyrios** [reply]
These scam artists do it because they can run google ads on the sites, and accept donations from users who don't realize they are giving to a rip off artist and not the official team.

You can see this on their donation pages:
https://archive.ph/Esbfr
https://archive.ph/EGSaw

Where donating users are giving but are unaware their donation is not going to who they intended.

**Reply by Ushie** [reply]
That's disgusting, I didn't know they were accepting donations

**Comment by Metadorius** [comment]
Suddenly now revanced.io is flagged by uBlock. Came here from the comments in the rules list to put some justice. While revanced.io looks like a "fake official website", they state in the faq that they are NOT an official website.

![image](https://github.com/revanced/revanced-discussions/assets/17500545/a4b2e8c4-ea88-41ab-9e7e-8e2b6634683d)

**Reply by oSumAtrIX** [reply]
Most people dont see it, its meant to look like the official site

**Reply by Metadorius** [reply]
I agree it would be better to clearly state on the front page that it's not an official website and it's a bit strange that it doesn't (I don't know the author intentions for sure but I don't think they're ill-formed, may be that he has some weird understanding of how it would be better to help ReVanced), but that still doesn't justify the "it's fake" claims. We must not distort the truth, and the truth is that it's an unofficial website without explicit claims to be official. People need to read the fine print more.

**Reply by Ushie** [reply]
We don't maintain the uBlock blocklist, we weren't involved in the process of getting it added to it

**Reply by Metadorius** [reply]
I never claimed you were. I am just leaving this bit of information here.

**Reply by oSumAtrIX** [reply]
Yes, it is an unofficial page and not more than that. Previously it was distributing prepatched apps, and everyone who experienced weird issues with their account reported to have sourced a prepatched apk from that site, hence why the accusations

**Reply by Metadorius** [reply]
That's understandable. What kind of weird issues by the way?

**Reply by LisoUseInAIKyrios** [reply]
Many of these fakes sites spread malicious apks.

As of this post, you can see this on http:ReVanced(.)to, by clicking on the upper right menu.  It opens a new window for a different site (myologyacetify(.)top) that google safe browsing identifies as malicious.  This malicious site then attempts to download some random non ReVanced related apk.

![Fake ReVanced site malware](https://github.com/ReVanced/revanced-discussions/assets/118716522/845b344b-ecc0-437f-94b4-0b3bc8916785)

**Reply by oSumAtrIX** [reply]
To add to this, advertisers had reached to us that wanted to implement similar popups on the website multiple times. Some have claimed they partnered with the known counterfeit sites for example. So far we have kindly refused to do that because such ads are impairing the user way too much, but counterfeits are profit oriented and thus do not care in that regard.
